README.md

```diff
@@ -98,136 +98,147 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
```

Comment:
That doc hasn't been updated in a year now. Should we really be linking to it??

Reply:
@ethanfangg  FYI
Thanks for pointing it out. Anyway, I guess that doesn't need to be part of a changelog update  😄

Reply:
https://github.com/microsoft/PowerToys/wiki/Roadmap